.npmrc

```diff
@@ -1,2 +1,4 @@
-registry=https://registry.npmjs.org/
 @yearn:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=${NPM_TOKEN}
```

Comment:
Could you keep that in your `~/.npmrc` root file? This appears to break vercel 🙏